content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-physics-setup.md

```diff
@@ -0,0 +1,145 @@
+---
+description: ' Setting up a ragdoll in the Open 3D Engine Animation Editor. '
+title: Setting up a Ragdoll
+---
+
+{{< important >}}
+There is currently no automatic way to prevent the physics geometry **PhysX Character Controller** component from colliding with the **PhysX Ragdoll** component, which can lead to unexpected behavior for the ragdoll. Collisions can be avoided by using [collision filtering](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/) or by disabling the physics on the **PhysX Character Controller** component when the ragdoll is activated.
+{{< /important >}}
+
+When you set up a ragdoll, you do the following:
++ [Define joints for the ragdoll.](#step-1-define-joints-for-the-ragdoll)
++ [Set up ragdoll colliders.](#step-2-set-up-ragdoll-colliders)
++ [Create joint limits.](#step-3-create-joint-limits)
+
+In addition, your actor must have a motion extraction node. Your ragdoll must have a root node that is a direct child of the motion extraction node. For example, the Rin character uses `root` for its motion extraction node. For the ragdoll root node, the Rin character uses `'C_pelvis_JNT`, which is a child node of `root`.
+
+While working on ragdoll setup, it can be useful to show or hide certain elements in the **Atom Render Window**. You can use the render options dropdown, represented by the ![Use the render options in the Atom Render Window to show or hide elements such as ragdoll colliders, joint limits and hit detection colliders](/images/user-guide/actor-animation/ragdoll-skeleton-render-options.svg) icon, to show or hide the **Ragdoll Colliders** (rendered in orange), **Ragdoll Joint Limits**, **Hit Detection Colliders** (rendered in blue), **Cloth Colliders** (rendered in purple), **Line Skeleton**, and other elements.
+
+## Step 1: Define Joints for the Ragdoll 
+
+Do the following to select the joints for your ragdoll. Usually you will want to include significantly fewer joints in the ragdoll than are present in the animation skeleton, as simulating small joints such as finger bones typically creates little benefit and incurs performance costs.
+
+**To select joints for the ragdoll**
+
+1. In O3DE Editor, choose **Tools**, **Animation Editor**.
+
+1. In the **Animation Editor**, on the right side of the menu bar, choose **Physics** from the drop-down list. This changes the layout.
+
+1. Choose **File**, **Open Actor** and select your actor.
+
+1. In the **Skeleton Outliner**, multi-select the joints that you want to include in your ragdoll. You can also select individual joints by left clicking on the **Line Skeleton** view in the **Atom Render Window**.
+
+1. Right-click one of the selected joints and then choose **Ragdoll**, **Add to ragdoll**.
+
+    {{< note >}}
+You can add joints to the ragdoll at any time.
+{{< /note >}}
+    {{< note >}}
```

Comment:
nit: indentation of {{<note>}} seems off here maybe?

Reply:
hultonha - I think you need the indentation to prevent it messing up the list. I've indented the closing braces as willihay suggested.